src/components/Scroll.js

```diff
@@ -8,6 +8,23 @@ import "../css/Scroll.scss";
  * Taken from https://reacttraining.com/react-router/web/guides/scroll-restoration
  */
 class Scroll extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { width: 0 };
+    this.updateWindowDimensions = this.updateWindowDimensions.bind(this);
+  }
+
+  componentDidMount() {
+    this.updateWindowDimensions();
+    window.addEventListener("resize", this.updateWindowDimensions);
+  }
+  componentWillUnmount() {
+    window.removeEventListener("resize", this.updateWindowDimensions);
+  }
+  updateWindowDimensions() {
```

Comment:
Add an empty line above this, and change it to an arrow function to avoid having to bind. Binding isn't "wrong", but will be more consistent to the rest of the project.